### DISCUSSION TITLE
Just wanted to say hi

### DISCUSSION THREAD

**Discussion by Bluscream** [community/community#132082] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I just wanted to give a warm Hi to everyone here! :wave:

**Comment by catunderrain** [comment]
Hi my friend! 🐇

**Comment by RachidZahrani** [comment]
(👉ﾟヮﾟ)👉

**Comment by ebndev** [comment, marked as answer]
Welcome to GitHub and the GitHub Community @Bluscream 🎆! We're glad you're here and can't wait to see where your GitHub journey takes you.

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and connect with other users there! 👋
